activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/test/java/org/activiti/cloud/services/identity/keycloak/KeycloakManagementServiceTest.java

```diff
@@ -59,6 +60,7 @@
 @ExtendWith(MockitoExtension.class)
 class KeycloakManagementServiceTest {
 
+    public static final String FAKE_GROUP = "fakeGroup";
```

Comment:
let's be more accurate with the wording:
`public static final String NON_EXISTENT_GROUP = "non-existent-group";`